src/commands/createFunction/createFunction.ts

```diff
@@ -49,20 +49,32 @@ export async function createFunctionFromCommand(
 export async function createFunctionInternal(context: IActionContext, options: api.ICreateFunctionOptions): Promise<void> {
     let workspaceFolder: WorkspaceFolder | undefined;
     let workspacePath: string | undefined = options.folderPath;
+
     if (workspacePath === undefined) {
-        workspaceFolder = await getWorkspaceFolder(context, options);
-        workspacePath = workspaceFolder.uri.fsPath;
+        workspaceFolder = await getWorkspaceFolder();
+        workspacePath = workspaceFolder ? workspaceFolder.uri.fsPath : undefined;
```

Comment:
Optional chaining is the best thing since sliced bread

```suggestion
        workspacePath = workspaceFolder?.uri.fsPath;
```

Reply:
Oh duh... 🤦‍♂️

Thanks for catching that!